snovault/tests/root.py

```diff
@@ -8,15 +8,21 @@
     Deny,
     Everyone,
 )
+from ..schema_utils import format_checker
 from ..calculated import calculated_property
 from ..resources import Root
 from ..config import root
 
 
-accession_re = re.compile(r'^SNO(SS|FL)[0-9][0-9][0-9][A-Z][A-Z][A-Z]$')
-test_accession_re = re.compile(r'^TST(SS|FL)[0-9][0-9][0-9]([0-9][0-9][0-9]|[A-Z][A-Z][A-Z])$')
+accession_re = re.compile(r'^SNO(SS|FL|AB)[0-9][0-9][0-9][A-Z][A-Z][A-Z]$')
+test_accession_re = re.compile(r'^TST(SS|FL|AB)[0-9][0-9][0-9]([0-9][0-9][0-9][0-9]|[A-Z][A-Z][A-Z][A-Z])$')
```

Comment:
Consider simplifying for easier reading (did this in smaht I think)
```suggestion
accession_re = re.compile(r'^SNO[A-Z]{2}[0-9]{3}[A-Z]{3}$')
test_accession_re = re.compile(r'^TST[A-Z]{2}[0-9]{3}([0-9]{4}|[A-Z]{4})$')
```